internal/common/compress/list.go

```diff
@@ -36,3 +44,11 @@ func DecompressStringArray(input []byte, decompressor Decompressor) ([]string, e
 	err = dec.Decode(&data)
 	return data, err
 }
+
+func MustDecompressStringArray(input []byte, compressor Decompressor) []string {
```

Comment:
Is called string but takes as input a `[]byte`.

Reply:
I'm assuming we're sure this'll never panic :)

Reply:
let me break this out into a separate test file so it can only be used by test code.

Reply:
actually no I won't because if I do that you can't import it from any other module!  I think it's ok as `MustX` is pretty idiomatic in go for something that will panic and should be used with great caution